### ISSUE TITLE
checkm data setRoot doesn't work on first execution

### ISSUE THREAD

**Issue by mw55309** [Ecogenomics/CheckM#148]
Hi

I have installed checkm using conda:

checkm-genome: 1.0.11-0 bioconda

This is what happens when I try and setRoot:

```sh
$ checkm data setRoot /home/ubuntu/checkm/
It seems that the CheckM data folder has not been set yet or has been removed. Running: 'checkm data setRoot'.
Where should CheckM store it's data?
Please specify a location or type 'abort' to stop trying:
```

As you can see, checkm immediately figures out it has not been run before, and begins the *manual* process to set the data root

Except I am using conda envs in an automated way through Snakemake and I need to set the data root automatically without any intervention.

How do I prevent checkm from entering the *manual* part of data setRoot?

Thanks
Mick

**Comment by abremges** [comment]
I ran into a similar issue and solved it by piping the path into setRoot, e.g.
```
checkm_db="/home/ubuntu/checkm/"
echo ${checkm_db} | checkm data setRoot ${checkm_db}
```

**Comment by cimendes** [comment]
Hello

I'm trying to create a docker container for checkM and i'm running into the same issue. Unfortunately @abremges solution did not work for me. Despite specifying a path in the setRoot argument, it still asks for it interactively. I find this behaviour very odd. Has anyone found a workaround? Thanks!

**Comment by abremges** [comment]
Did you also pipe the path string into `checkm data setRoot`? See e.g. https://hub.docker.com/r/virusx/checkm/~/dockerfile/

(This specific Docker container expects the CheckM data to be mounted to `/var/spool/cwl/checkm_data_v1.0.7/`)

**Comment by cimendes** [comment]
I must have been doing some mistake somewhere cause now it works. Thanks!!!

**Comment by dparks1134** [comment]
This should be resolved in v1.0.15 which I plan to release later today.
